templates/_base/footer.html

```diff
@@ -11,6 +11,9 @@
         <li class="p-inline-list__item">
           <a class="p-link--soft" href="https://github.com/canonical-websites/jp.ubuntu.com/issues/new" id="report-a-bug">Report a bug on this site</a>
         </li>
+        <li class="p-inline-list__item">
```

Comment:
I think Bartek was right, this would be better before the 'report a bug'... link


What do you think?

Reply:
Also, I am ready with GTM when we release.

Reply:
Done.